doc/src/plugins/soln-plugin-tavg.rst

```diff
@@ -0,0 +1,103 @@
+******************
+[soln-plugin-tavg]
+******************
+
+Time average quantities. Parameterised with
```

Comment:
Add a couple of paragraphs with more detail around usage (windowing modes, getting fluctuations, ...)